samples/excel/30-events/events-worksheet.yaml

```diff
@@ -0,0 +1,202 @@
+order: 8
+id: excel-events-worksheet
+name: Worksheet events
+description: 'Registers event handlers for selection, data, and calculation changes.'
```

Comment:
For consistency with other snippet descriptions, change to -> `Registers event handlers that run when data is changed in worksheet, the selected range changes in a worksheet, or the worksheet is recalculated.`